bitnami/wordpress/templates/ingress.yaml

```diff
@@ -21,7 +23,7 @@ spec:
   {{- end }}
   rules:
     {{- if .Values.ingress.hostname }}
-    - host: {{ .Values.ingress.hostname | quote }}
+    - host: {{ tpl .Values.ingress.hostname . | quote }}
```

Comment:
```suggestion
    - host: {{ tpl .Values.ingress.hostname $ | quote }}
```

Reply:
Why would you use $ here? . does the same, right? It is not in a loop, so it should do the same?